CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Embedded Graphics is a `no_std` library for adding graphics features to display
 
 ## [Unreleased] - ReleaseDate
 
+### Fixed 
+
+- [#734](https://github.com/embedded-graphics/embedded-graphics/pull/734) Fix panic if the README example is ran
+
```

Comment:
I don't think this change needs a changelog entry. Our policy had been to only include important API changes in the changelog to make it easier for users to find the relevant changes.